### PR TITLE
Added cypress tests for tags, users to follow, contributors and upcoming events

### LINKED ISSUE #327: Add cypress tests for card tabs

### PR DESCRIPTION
**Pull request by Janvi-Thakkar**
## Related Issue
Fixes #327 

## Screenshots or GIF (In case of UI changes):
https://user-images.githubusercontent.com/55620053/177396564-1e3c47c3-e00d-4c04-815f-c98efc9fafe6.mp4

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Niweera** [comment]
seems like you haven't run prettier on some of the js files. can you double check?

**Comment by Janvi-Thakkar** [comment]
> seems like you haven't run prettier on some of the js files. can you double check?

I have run prettier for every file before pushing it.

File: src/components/HomePage/index.js [hunk 1]
```diff
@@ -12,103 +12,168 @@ import List from "@material-ui/core/List";
...
-        { name: "Privacy Policy", link: "https://dev.codelabz.io/" },
-        { name: "CodeLabz @2022", link: "https://dev.codelabz.io/" },
+  const [value, setValue] = useState(2);
+  const [footerContent, setFooterContent] = useState([
+    { name: "Help", link: "https://dev.codelabz.io/" },
+    { name: "About", link: "https://dev.codelabz.io/" },
+    { name: "Content Policy", link: "https://dev.codelabz.io/" },
+    { name: "Terms", link: "https://dev.codelabz.io/" },
+    { name: "Privacy Policy", link: "https://dev.codelabz.io/" },
+    { name: "CodeLabz @2022", link: "https://dev.codelabz.io/" },
```
Comment (Niweera) [inline src/components/HomePage/index.js:29]: shall we change this to 
```
name: `CodeLabz @${new Date().getFullYear()}`
```

in this way the year will be changed accordingly.

**Comment by Niweera** [comment]
> > seems like you haven't run prettier on some of the js files. can you double check?
> 
> I have run prettier for every file before pushing it.

sure no problem

File: src/components/HomePage/index.js (hunk 1 above)
Comment (Janvi-Thakkar) [reply src/components/HomePage/index.js:29]: @Niweera sure, I will do it.

**Review by Niweera — APPROVED** [review]

**Review by AsithaIndrajith — APPROVED** [review]

**Pull request merged**